_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,425 @@
+    display: inline-block;
+    margin-left: ($baseline/4) + px;
+    content: "*";
+    font-size: font-size(small);
+    color: $required-indicator-color;
+}
+
+.input,
+.select,
+.textarea {
+    @include transition(all timing(xx-fast) ease-in-out 0s);
+    display: inline-block;
+    min-width: ($baseline*20) + px; // for the purposes of the demo... can probably get moved elsewhere...
+    padding: ($baseline/2) + px;
+    border: 1px solid $field-default-border;
+    background: $field-default-background;
+    font-family: $font-family-sans-serif;
+    font-size: font-size(base);
+    color: $field-default-text-color;
+
+    &:focus {
+        outline: none;
+    }
+}
+
+.progress {
+    @include transition(all timing(xx-fast) ease-in-out 0s);
+}
+
+
+::-webkit-input-placeholder,
+::-moz-placeholder,
+:-moz-placeholder,
+:-ms-input-placeholder {
+   color: $placeholder-text-color;
+}
+
+// ----------------------------
+// #INPUT TEXT
+// ----------------------------
+.input-text {
+
+    &:focus,
+    &.is-active {
+        border-color: $field-default-border-focus;
+        color: $field-default-text-color-hover;
+    }
+
+    &:disabled,
+    &.is-disabled {
+        border-color: $field-disabled-border;
+        background: $field-disabled-background;
+    }
+
+    &.input-simple {
+        padding: ($baseline/2) + px 0;
+        border-width: 0 0 2px 0;
+
+        &:focus,
+        &.is-active {
+            border-color: $field-simple-border-focus;
+            background: $field-simple-background-focus;
+        }
+    }
+}
+
+// ----------------------------
+// #SELECT
+// ----------------------------
+.wrapper-replace-select {
+    position: relative;
+}
+
+.input-select {
+    height: ($baseline*2) + px;
+    line-height: ($baseline*2) + px;
+
+    &:focus,
+    &.is-active {
+        border-color: $field-default-border-focus;
+        color: $field-default-text-color-hover;
+    }
+
+    &:disabled,
+    &.is-disabled {
+        border-color: $field-disabled-border;
+        background: $field-disabled-background;
+    }
+
+    &.is-replaced {
+        position: absolute;
+        z-index: z-index(mid-front);
+    }
+
+    &.is-transparent {
+        opacity: 0;
+    }
+}
+
+.wrapper-custom-select {
+    @extend .input;
+    @extend .input-select;
+    position: absolute;
+    display: inline;
+    top: 0;
+    left: 0;
+    width: auto;
+    line-height: 100%;
+    z-index: z-index(base);
+
+    svg {
+        position: absolute;
+        right: ($baseline/2) + px;
+        height: $baseline + px;
+        width: $baseline + px;
+        fill: $field-default-border;
+    }
+
+    &:hover,
+    &:focus,
+    &.is-hover,
+    &.is-active {
+        border: 1px solid $field-default-border-focus;
+
+        svg {
+            @include transition(all timing(xx-fast) ease-in-out 0s);
+            fill: $field-default-border-focus;
+        }
+    }
+
+    &:disabled,
+    &.is-disabled {
+        cursor: default;
```

Comment:
Good question. I've seen both `default` and `not-allowed`, but I think the latter makes more sense here. We should set a standard, rather than allow something like this to be themed. Can't make it easy to make trailing mouse cursors, after all.